src/components/common/ActionModal.vue

```diff
@@ -262,6 +268,16 @@ export default {
       }
       return signWithLocalKeystore
     },
+    checkBrowser() {
```

Comment:
i might call this `browserWithLedgerSupport` or something a little bit more descriptive like that.